assets/js/auth.js

```diff
@@ -1,9 +1,9 @@
-function checkEmailPattern(email) {
+function checkEmail(email) {
   const emailPattern = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/i;
-  return emailPattern.test(email) ? true : false;
+  return emailPattern.test(email);
 }
 
-function checkPasswordPattern(password) {
+function checkPassword(password) {
   const passwordPattern = /^(?=.*[a-zA-Z])(?=.*[0-9]).{8,25}$/;
   return passwordPattern.test(password) ? true : false;
```

Comment:
`passwordPattern.test(password)` 자체로 boolean 을 반환하니 아래와 같이 변경 가능합니다 (checkEmail 함수에서 작성한 방식과 통일)

```suggestion
  return passwordPattern.test(password);
```